app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command_name, file_name, destination_path = command.split()
+
+        if "/" in destination_path:
+            destination_path = destination_path.split("/")
+            new_file_name = destination_path[-1]
+            destination_path = destination_path[:-1]
+            for path in destination_path:
+                os.makedirs(path)
```

Comment:
makedirs should be used to create directories without using loop, you can just pass the full path (exist_ok=True argument should be passed additionally to avoid os errors if the directory already exists)